Dockerfile

```diff
@@ -2,6 +2,7 @@ FROM alpine:3.10.1
 RUN apk update \
   && apk add --no-cache fio==3.13-r1 ca-certificates wget \
   && update-ca-certificates
+VOLUME /app
```

Comment:
Did not you mean to provide volume for benchmarking?
If so, we should create a dedicated volume for that and leave app untouched.

Reply:
Actually, we need a volume for the working directory (where fio does the read/write tests). In this case it happens to be the same directory, but I will create a separate one for it for clarity.